modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordExecManager.kt

```diff
@@ -9,6 +9,27 @@ import com.intellij.openapi.util.SystemInfo
 import com.intellij.openapi.vfs.VirtualFileManager
 import java.nio.file.Path
 
+/**
+ * # Guide on how to build and run the plugin
+ *
+ * To build just the mirrord plugin, do:
+ *
+ * `./gradlew buildPlugin`
+ *
+ * If you want to also run tests + lint + build the plugin:
+ *
+ * `./gradlew build`
+ *
+ * The minimum java version required is 17, if you get some `inputReader` error while building the plugin, that's
+ * probably why (outdated java version), or you may need to clean the project (search for Intellij `Invalidate cache`
+ * option).
+ *
+ * With the plugin built, you'll now have a folder `mirrord-intellij/build/distributions` that contains the
+ * `mirrord-x.y.z.zip` plugin. To install it, go on the `Plugins` option in Intellij, remove the mirrord plugin (if you
+ * already had it installed), then click on the "Settings icon" and select "Install from disk". Pick the
+ * `mirrord-x.y.z.zip` you just built, and that's it (you'll probably have to restart the IDE).
+ */
```

Comment:
we already have a section in CONTRIBUTING.md, so i think we should just add the extra info to that and remove this

Reply:
Oh snap, that's what I get from only searching for `.kt` (and source files) when opening folders.

Will remove.